src/components/atomic/Loading/Loading.tsx

```diff
@@ -9,7 +9,7 @@ type LoadingProps = {
 const Loading: React.FunctionComponent<LoadingProps> = ({ sx = {} }) => {
   return (
     <Box sx={{ height: 40, width: 40, ...sx }}>
-      <img src="/images/loading.gif" alt="image" height="100%" width="100%" />
+      <img src="/images/loading.gif" height="100%" width="100%" />
```

Comment:
do we need an altprop in here as suggested by gh actions?

Reply:
I think gh actions commented that we don't need alt for this case.

Reply:
fixed in: 8fe2861f0d791155669d7d1c7cb0988318af25c8